sport/app/cricket/feed/cricketPaFeed.scala

```diff
@@ -52,38 +52,36 @@ class PaFeed(wsClient: WSClient, actorSystem: ActorSystem, materializer: Materia
     }
   }
 
-  def getMatchIds(team: CricketTeam)(implicit executionContext: ExecutionContext): Future[Seq[String]] = {
-
-    // Get fixtures for England for today.
-    val fixtures = getTeamMatches(team, "fixtures", LocalDate.now, LocalDate.now)
-
-    // Get results for England over the last year.
-    val results = getTeamMatches(team, "results", LocalDate.now.minusMonths(2), LocalDate.now)
+  def getCurrentMatchIds(team:CricketTeam)(implicit executionContext: ExecutionContext): Future[Seq[String]] = {
+    getTeamMatches(team, "fixtures", LocalDate.now, LocalDate.now)
```

Comment:
I think it's possible there's some confusion here about the difference between fixtures, results, and today's matches. I don't know the Cricket API, but I'd recommend having a read of the docs to check what the terminologies are. It may be (for example) that in-progress games are treated as results (just results that aren't yet finished). In that case this wouldn't be picking them up.

Reply:
Absolutely. Thanks for spotting this